.env.development

```diff
@@ -0,0 +1 @@
+BACKEND_URL=http://35.193.167.21
```

Comment:
Is this the ip address of the backend in the dev cluster or locally?

Reply:
dev cluster